packaging/bundle-validation/ci_run.sh

```diff
@@ -145,27 +147,33 @@ if [ -z "$STAGING_REPO_NUM" ] && [ -z "$MAVEN_BASE_URL" ]; then
 else
   echo 'Adding environment variables for bundles in the release candidate or artifact'
 
+  HUDI_CLI_BUNDLE_NAME=hudi-cli-bundle
```

Comment:
Similarly, we can get rid of this line as the bundle name is scala version specific and this line is not used (i.e., the env var is reassigned in the if-else branch).